Translations/de.lproj/Localizable.strings

```diff
@@ -215,20 +215,20 @@ The language used in MAS/AppleID will be used in Keka's tip strings and may not
 Note that the short string is limited to 30 characters and the long string is limited to 45 characters.
 */
 "Monthly sip of coffee" = "Monthly sip of coffee";
```

Comment:
This one is missing if you can also translate it :)

Reply:
Done 👍  and exactly 30 characters.